### DISCUSSION TITLE
Add hotkey to enable/disable RTX, rather than menu

### DISCUSSION THREAD

**Discussion by notacodeman** [sultim-t/xash-rt#115] [Ideas]
Please add an option to bind a key to quickly enable/disable the raytracing.
Or if there is a console command we can bind, please let us know!

**Comment by saarstar15** [comment]
The "x" key works for me by default.

**Reply by AnthonyPac** [reply]
x does nothing for me

**Comment by VanderCat** [comment]
add `bind F4 rt_classic_toggle` to config.cfg or evaluate in console (of course f4 can be any button you want)
